portal/src/locale-data/en.json

```diff
@@ -1016,7 +1016,7 @@
 
   "WizardContentLayout.skip.label": "Skip and I’ll set up my project manually",
 
-  "CreateProjectScreen.title": "Create your first project",
+  "CreateProjectScreen.title": "{apps, plural, zero{Create your first project} other{Create a new project}}",
```

Comment:
Seems `zero` is not supported. Maybe we should change the copy to `{apps, plural, =0{Create your first project} other{Create a new project}}`?

Reply:
English recognizes `one` and `other` only. So you're right, we have to use explicit value here.